### DISCUSSION TITLE
[Support] 在 Guard 中如何判断 Metadata？

### DISCUSSION THREAD

**Discussion by hooliy** [midwayjs/midway#3719] [Q&A]
### Describe the problem（描述问题）

![image](https://github.com/midwayjs/midway/assets/3368691/462f8977-965e-425f-80de-981440d0e2d3)
![image](https://github.com/midwayjs/midway/assets/3368691/34963056-bca8-40fb-86ab-775f19ae4708)

**定义了装饰器，在守卫中如何判断是否有此装饰器？**

像在Nestjs中可以这样实现：
![image](https://github.com/midwayjs/midway/assets/3368691/8cb2a6be-2924-4f41-9723-7bff9045d826)

### Midway Versions（Midway 版本）

latest

**Comment by wjw-gavin** [comment]
请问实现了吗？

**Reply by hooliy** [reply]
自定义装饰器的时候传true就可以了. 不过我换用nestjs了
